src/hooks/useGasFee.ts

```diff
@@ -78,9 +81,10 @@ const useGasFee = (selectedToken, needApproval) => {
           setMaxFeePerGas(null)
           setMaxPriorityFeePerGas(null)
           setError(trimErrorMessage(error.message))
+          throw error
         })
-    },
-  })
+  }, [blockNumber, needApproval, priceFeeGasPrie, priceFeeGasLimit, instance])
```

Comment:
when executing `calculateGasFee`, `priceFeeGasPrie(should be priceFeeGasPrice)` and `priceFeeGasLimit` are not used. Why do they need to be included in the dependencies?

Reply:
How about using `useCallback` to establish dependency relationships? I'm concerned that current code might lead to confusion in future modifications.